skrf/network.py

```diff
@@ -5482,6 +5482,7 @@ def f(x, y):
     ntwkC = ntwkA.copy()
     ntwkC.s = C
     ntwkC.z0 = np.hstack([ntwkA.z0, ntwkB.z0])
+    ntwkC.port_modes = np.hstack([ntwkA.port_modes, ntwkB.port_modes])
```

Comment:
Can you also fix the missing spaces in the docstring ? I believe you should only change

``` 
>>>concat([ntwkA,ntwkB])
>>>concat([ntwkA,ntwkB,ntwkC,ntwkD], port_order='second')
To put for lines in parallel
>>> from skrf import air
>>> l1 = air.line(100, z0=[0,1])
>>> l2 = air.line(300, z0=[2,3])
>>> l3 = air.line(400, z0=[4,5])
>>> l4 = air.line(400, z0=[6,7])
>>> concat_ports([l1,l2,l3,l4], port_order='second')
```

to

```
>>> concat([ntwkA,ntwkB])
>>> concat([ntwkA,ntwkB,ntwkC,ntwkD], port_order='second')

To put for lines in parallel

>>> from skrf import air
>>> l1 = air.line(100, z0=[0,1])
>>> l2 = air.line(300, z0=[2,3])
>>> l3 = air.line(400, z0=[4,5])
>>> l4 = air.line(400, z0=[6,7])
>>> concat_ports([l1,l2,l3,l4], port_order='second')
```

Reply:
Changes made as requested.